scripts/makeupdates

```diff
@@ -27,6 +27,9 @@ import six
 import subprocess
 import sys
 
+# The Python site-packages path for blivet.
+PYTHON_LIB_PATH = "./usr/lib/python3.9/site-packages/"
```

Comment:
I don't like this. This is forcing us to update this path for every Fedora release (don't forget we don't tie our releases to Fedora releases) and also makes the updates images version specific -- and not Fedora version specific but Python version specific which changes **during** rawhide development -- most extreme example could be updates image working with latest rawhide release and not the one from day before.

Reply:
If we really must have "version specific" updates images, it would make more sense (for us) to always build RPMs here for a specific Fedora version in mock and create updates image from them.